### DISCUSSION TITLE
Wishlist Functionality

### DISCUSSION THREAD

**Discussion by Adam-Mould** [saleor/saleor#8467] [Q&A] [answered]
I can see on the roadmap that Wishlists are planned under "Future releases", however after browsing the saleor code there has been some work done around the models, and some GraphQL work (mutations and some resolvers).

With that said, is the bulk of the remaining wishlist functionality mainly around finishing off the GraphQL queries, and integrating it into the demo storefront?

I'm looking to add wishlist functionality to a custom eCommerce store I'm currently building with Saleor, but unsure whether I could leverage the underlying wishlist API at present.

Any thoughts would be much appreciated.

**Comment by karolkielecki** [comment, marked as answer]
@Adam-Mould Hi! Wishlists require much more work maybe even from scratch--it's a big feature. There is no ETA for wishlists. If there are any changes, we'll update our roadmap. If I have any news, I'll also keep you posted.

Regarding the storefront, please check the status of the storefront app: https://github.com/saleor/saleor/discussions/8383#discussioncomment-1503509

**Reply by Adam-Mould** [reply]
That's great, thank you for your response.

For now, I'll implement a basic wishlist functionality another way :)
